content/developer/reference/backend/orm/changelog.rst

```diff
@@ -11,3 +11,12 @@ Odoo Online version 15.2
   `#83015 <https://github.com/odoo/odoo/pull/83015>`_, developers can now define what type of
   indexes can be used on fields by PostgreSQL. See the :ref:`index property <reference/fields>` of
   `odoo.fields.Field`.
+
+Odoo Online version 15.4
```

Comment:
I'd swap 15.2 and 15.4 to have the latest version on the top. In a few years, we'll want to directly see (without scrolling all the way down to the bottom of the page) what changed in Odoo 19.2, not 15.2.